ui-tests/src/test/java/org/openmrs/reference/page/OpenConceptLabPage.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class OpenConceptLabPage extends Page {
+    private static final By SUBSCRIPTION_URL = By.id("subscription-url");
+    private static final By SETUP_SUBSCRIPTION_BUTTON = By.cssSelector("#body-wrapper > home > div:nth-child(2) > div:nth-child(2) > a");
+    private static final String CONCEPT_URL = "https://qa-refapp.openmrs.org/openmrs/dictionary/concept.htm?conceptId=5258";
+    private static final String TOKEN_URL = "https://qa-refapp.openmrs.org/openmrs/dictionary/concept.htm?conceptId=5258";
+    private static final By SAVE_BUTTON = By.cssSelector("#body-wrapper > subscription > div:nth-child(1) > form > fieldset > div:nth-child(4) > button:nth-child(1)");
+    private static final By CANCEL_BUTTON = By.cssSelector("#body-wrapper > subscription > div:nth-child(1) > form > fieldset > div:nth-child(4) > button:nth-child(2)");
```

Comment:
even for CANCEL_BUTTON !

Reply:
@kdaud kindly on the css selector is a button won't it  create an error when I change to `Id` Instead?

Reply:
@jwnasambu what happens when you use id() instead of cssSelector() in reference those page elements?